.github/workflows/build.yaml

```diff
@@ -0,0 +1,43 @@
+name: '🏗️ Build jellyfin-ffmpeg'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - jellyfin
+    paths-ignore:
+      - '**/*.md'
+
+  pull_request:
+    branches:
+      - jellyfin
+    paths-ignore:
+      - '**/*.md'
+
+jobs:
+  build_debian:
+    uses: jellyfin/jellyfin-ffmpeg/.github/workflows/_meta.yaml@master
+    with:
+      distro: 'debian'
+      codenames: "[\"buster\", \"bullseye\"]"
```

Comment:
could you not use `'["buster", "bullseye"]'` instead of the escape? would look cleaner IMHO if possible

Reply:
I took this from here where it was explictily escaped:
https://github.community/t/reusable-workflow-with-strategy-matrix/205676